tests/databases/test_sqlite.py

```diff
@@ -120,6 +120,52 @@ def test_sqlite_create_table_with_columns(database_table_fixture):
     assert rows[1] == (1, "name", "VARCHAR(60)", 1, None, 0)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [{"database": Database.SQLITE, "table": Table()}],
+    indirect=True,
+    ids=["sqlite"],
+)
+def test_sqlite_create_table_autodetection_with_file(database_table_fixture):
```

Comment:
Since the `BaseDatabase` is an abstract class and can't be instantiated, a specific implementation is the only way to test these functionalities. Since we are having to use a specific implementation, I placed it in the SQLlite test. This pattern already existed before, eg. for the previous `create_table` and `run_sql`, and it makes clear which tests rely on this particular Database.